packages/actor-rdf-join-wrap-stream/README.md

```diff
@@ -0,0 +1,40 @@
+# Comunica Wrap Stream RDF Join Actor
+
+[![npm version](https://badge.fury.io/js/%40comunica%2Factor-rdf-join-wrap-stream.svg)](https://www.npmjs.com/package/@comunica/actor-rdf-join-wrap-stream)
+
+A comunica Wrap Stream RDF Join Actor. This actor calls the `bus-iterator-transform` for all iterators produced by `query-operation` actors.
+This actor should only be included if you require the functionality of `bus-iterator-transform` as it will likely slow down query execution.
```

Comment:
```suggestion
This actor should only be included if you require the functionality of `bus-iterator-transform` as it may slow down query execution.
```